tests/jax/components/training/step_test.py

```diff
@@ -0,0 +1,723 @@
+                "agent_0": array([[0.0, 0.0, 0.0], [0.0, 0.0, 0.0]], dtype=float32),
+                "agent_1": array(
+                    [
+                        [0.0897511, 0.14813632, 0.19152136],
+                        [0.06387269, 0.00243677, 0.0],
+                    ],
+                    dtype=float32,
+                ),
+                "agent_2": array(
+                    [[0.0, 0.0, 0.0], [0.0897511, 0.14813632, 0.19152136]],
+                    dtype=float32,
+                ),
+            },
+            discounts={
+                "agent_0": array([[1.0, 1.0, 1.0], [1.0, 1.0, 1.0]], dtype=float32),
+                "agent_1": array([[1.0, 1.0, 1.0], [1.0, 1.0, 1.0]], dtype=float32),
+                "agent_2": array([[1.0, 1.0, 1.0], [1.0, 1.0, 1.0]], dtype=float32),
+            },
+            start_of_episode=array([[True, False, False], [False, False, False]]),
+            extras={
+                "network_keys": {
+                    "agent_0": array([[0, 0, 0], [0, 0, 0]], dtype=int32),
+                    "agent_1": array([[0, 0, 0], [0, 0, 0]], dtype=int32),
+                    "agent_2": array([[0, 0, 0], [0, 0, 0]], dtype=int32),
+                },
+                "policy_info": {
+                    "agent_0": array(
+                        [
+                            [-1.5010276, -1.5574824, -1.7098966],
+                            [-1.6839617, -1.8447837, -1.4597069],
+                        ],
+                        dtype=float32,
+                    ),
+                    "agent_1": array(
+                        [
+                            [-1.6333038, -1.7833046, -1.4482918],
+                            [-1.6957064, -1.4800832, -1.66526],
+                        ],
+                        dtype=float32,
+                    ),
+                    "agent_2": array(
+                        [
+                            [-1.4521754, -1.4560769, -1.8592778],
+                            [-1.4220893, -1.78906, -1.5569873],
+                        ],
+                        dtype=float32,
+                    ),
+                },
+            },
+        )
+
+
+@pytest.fixture
+def mock_trainer() -> MockTrainer:
+    """Build fixture from MockTrainer"""
+    return MockTrainer()
+
+
+@pytest.fixture
+def dummy_sample() -> DummySample:
+    """Build fixture from DummySample"""
+    return DummySample()
+
+
+def test_default_trainer_step_initiator() -> None:
+    """Test constructor of DefaultTrainerStep component"""
+    trainer_step = DefaultTrainerStep()
+    assert trainer_step.config.random_key == 42
+
+
+def test_on_training_step_with_timestamp(mock_trainer: Trainer) -> None:
+    """Test on_training_step method from TrainerStep case of existing timestamp"""
+    trainer_step = DefaultTrainerStep()
+    old_timestamp = mock_trainer.store.timestamp
+    trainer_step.on_training_step(trainer=mock_trainer)
+
+    assert int(mock_trainer.store.timestamp) == int(time.time())
```

Comment:
Is there not a way to rather just check that the timestamp value has increased? I am concerned that the test might break due the `time.time()` somehow not being able to align with the timestamp.

Reply:
Another solution could be just to check it is bigger then old_timestamp, I will try to make it instead of time.time

Reply:
Sounds good.